### DISCUSSION TITLE
Image getting scaled to center crop when I added transformation to it

### DISCUSSION THREAD

**Discussion by Krushikar** [coil-kt/coil#1317] [General]
Whenever I added transformation to image it getting scaled to center crop. I had tried setting scale type to FIT and FITXY by
programmatically and by XML but none of the method works.

I want to scale the image to fit imageview and want to set the rounded corner for image, but whenever I tried to add transformation to the image it's getting scaled to center crop.

library version 
implementation("io.coil-kt:coil:2.0.0-rc02")

**Comment by ITAnt** [comment]
Any solutions?
